db/seeds.rb

```diff
@@ -1,7 +1,11 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+# TODO: Remove network and school seeding when we have a way of getting them from GIAS
+
+network = Network.create!(name: "Test school network")
```

Comment:
Is this idempotent with the `!`? We want to be able to run this multiple times

Reply:
Hmmm, good point. I think `create` and `create!` only differ in what happens when validation fails.

So really, if we want them idempotent we would probably need to use `find_or_create_by`. But I am not sure we would want to run seeding multiple times without deleting the data in between - when would that happen?